src/content/nl/2020/third-parties.md

```diff
@@ -0,0 +1,298 @@
+title: Derden
+description: Hoofdstuk Derden van de Web Almanac 2020 met informatie over welke derden worden gebruikt, waarvoor ze worden gebruikt, impact op de prestaties en impact op de privacy.
+authors: [simonhearne]
+reviewers: [jzyang, exterkamp]
+analysts: [max-ostapenko, paulcalvano]
+editors: [bazzadp]
+translators: [noah-vdv]
+simonhearne_bio: Simon is een webprestatie-architect. Hij heeft een passie voor het helpen leveren van een sneller en toegankelijker web. Je kunt hem vinden op twitter<a href="https://twitter.com/simonhearne">@SimonHearne</a> en zijn blog <a hreflang="en" href="https://simonhearne.com">simonhearne.com</a>.
+discuss: 2042
+results: https://docs.google.com/spreadsheets/d/1uW4SMkC45b4EbC4JV1xKAUhwGN2K8j0qFy_jSIYnHhI/
+featured_quote: Inhoud van derden komt vaker dan ooit voor, 94% van de pagina's heeft ten minste één bron van derden en de mediaanpagina heeft 24. In dit hoofdstuk bespreken we de prevalentie van inhoud van derden, de impact op het paginagewicht en de browser-CPU, en stellen vervolgens methoden voor om de impact van inhoud van derden op de paginaprestaties te verminderen.
+featured_stat_1: 94,1%
+featured_stat_label_1: Pagina's met inhoud van derden
+featured_stat_2: 21,5%
+featured_stat_label_2: Inhoud van derden geleverd als JavaScript
+featured_stat_3: 24
+featured_stat_label_3: Verzoeken van derden op de mediaanpagina
+---
+
+## Inleiding
+
+Inhoud van derden is tegenwoordig een essentieel onderdeel van de meeste websites. Het ondersteunt alles: analyse, livechat, advertenties, video's delen en meer. Content van derden biedt waarde door het zware werk van site-eigenaren uit handen te nemen en stelt hen in staat zich te concentreren op hun kerncompetenties.
+
+Velen beschouwen inhoud van derden als JavaScript-gebaseerd, maar de gegevens laten zien dat dit slechts geldt voor 22% van de verzoeken. Inhoud van derden is er in alle vormen, van afbeeldingen (37%) tot audio (0,1%).
+
+In dit hoofdstuk bespreken we de prevalentie van inhoud van derden en hoe dit is veranderd sinds 2019. We zullen ook bekijken: de impact van inhoud van derden op het paginagewicht (een goede proxy voor de algehele impact op de prestaties), scripts die vroeg worden geladen in de paginalevenscyclus, de impact van inhoud van derden op de CPU-tijd van de browser en hoe open derden zijn met hun prestatiegegevens.
+
+## Definities
+
+Voordat we op de gegevens ingaan, moeten we de terminologie definiëren die in dit hoofdstuk wordt gebruikt.
+
+### "Derde partij"
+
+Een bron van derden is een entiteit buiten de primaire site-gebruikerrelatie. Het betreft de aspecten van de site die niet rechtstreeks onder de controle van de site-eigenaar vallen, maar aanwezig zijn, met hun goedkeuring. Het Google Analytics-script is bijvoorbeeld een veelgebruikte bron van derden.
+
+We beschouwen bronnen van derden als:
+
+* Gehost op een _shared_ en _public_ oorsprong
```

Comment:
"shared" and "public" aren't technical terms here, but are only in italics for emphasis (and not even sure they should be emphasised to be honest, but that's another issue).

So I think they should be translated:

```suggestion
* Gehost op een _ gedeelde_ en _ openbaar_ oorsprong
```